[2022-01-11 18:53:55,511] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B scheduled__2022-01-10T00:00:00+00:00 [queued]>
[2022-01-11 18:53:55,529] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B scheduled__2022-01-10T00:00:00+00:00 [queued]>
[2022-01-11 18:53:55,529] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 18:53:55,529] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-01-11 18:53:55,529] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 18:53:55,792] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): training_model_B> on 2022-01-10 00:00:00+00:00
[2022-01-11 18:53:55,795] {standard_task_runner.py:52} INFO - Started process 4672 to run task
[2022-01-11 18:53:55,839] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_B', 'scheduled__2022-01-10T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpjgvzn886', '--error-file', '/tmp/tmpdyukjgrh']
[2022-01-11 18:53:55,840] {standard_task_runner.py:77} INFO - Job 18: Subtask training_model_B
[2022-01-11 18:53:56,248] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_B scheduled__2022-01-10T00:00:00+00:00 [running]> on host 40c4c8725776
[2022-01-11 18:53:56,506] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-01-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-10T00:00:00+00:00
[2022-01-11 18:53:56,506] {python.py:175} INFO - Done. Returned value was: 6
[2022-01-11 18:53:56,664] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_B, execution_date=20220110T000000, start_date=20220111T185355, end_date=20220111T185356
[2022-01-11 18:53:56,809] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 18:53:57,112] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
